model/span.go

```diff
@@ -189,6 +197,19 @@ func (d *DestinationService) fields() common.MapStr {
 	return common.MapStr(fields)
 }
 
+func (c *Composite) fields() common.MapStr {
+	if c == nil {
+		return nil
+	}
+	var fields mapStr
+	if c.Count > 1 {
+		fields.set("count", c.Count)
+	}
+	fields.maybeSetBool("exact_match", &c.ExactMatch)
```

Comment:
This only sets the field when it's true. Is that intentional, or should it always be set?

Reply:
It looks like from [this code](https://github.com/elastic/apm-server/blob/master/model/mapstr.go#L39-L45) that the boolean value is set as long as it's not `nil`. Am I reading the code incorrectly?
Also, [this test](https://github.com/elastic/apm-server/pull/5661/files#diff-69fa71f505412f98fb040c0923f0ff552c47f73e3315356a26b1eb41938a9a71R123-R165) would fail if it only set the field when it was true.

Reply:
Oh sorry, I missed that `c.ExactMatch` is a bool field, and you're taking the address of it. Since you're taking the address, and the pointer is always non-nil, it will always be set. In that case, this would be simpler:

```suggestion
	fields.set("exact_match", c.ExactMatch)
```

Reply:
That makes sense, thanks!